backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/snippet/internal/BpkSnippetImpl.kt

```diff
@@ -0,0 +1,103 @@
+/**
+ * Backpack for Android - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.skyscanner.backpack.compose.snippet.internal
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.aspectRatio
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import net.skyscanner.backpack.compose.snippet.ImageOrientation
+import net.skyscanner.backpack.compose.text.BpkText
+import net.skyscanner.backpack.compose.theme.BpkTheme
+import net.skyscanner.backpack.compose.tokens.BpkBorderRadius
+import net.skyscanner.backpack.compose.tokens.BpkSpacing
+import net.skyscanner.backpack.compose.utils.clickable
+
+@Composable
+internal fun BpkSnippetImpl(
+    content: @Composable (() -> Unit),
+    imageOrientation: ImageOrientation,
+    headline: String?,
+    description: String?,
+    bodyText: String?,
+    modifier: Modifier = Modifier,
+    onClick: (() -> Unit)?,
+) {
+    val roundedCornerShape = RoundedCornerShape(BpkBorderRadius.Sm)
+    Column(
+        modifier = modifier
+            .apply {
+                padding(horizontal = BpkSpacing.Lg)
+                if (onClick != null) {
+                    clickable(onClick = onClick)
+                }
+            },
+    ) {
+        Box(
+            modifier = Modifier
+                .padding(bottom = BpkSpacing.Base)
```

Comment:
This is the only point I'm not sure because iOS is not adding this padding here.
I'll check with the design team 😄

Reply:
https://skyscanner.slack.com/archives/C04FCMBSZLK/p1691143786967049

Reply:
I see if iOS didn't apply it I would do the same. It doesn't really make sense for there to be padding if it's just the image - it is just the spacer to the text, so should probably be included there instead